components/ActivityIndicator/src/MDCActivityIndicator.m

```diff
@@ -780,6 +784,19 @@ + (CGFloat)defaultHeight {
   return kSpinnerRadius * 2.f;
 }
 
++ (NSArray<UIColor *> *)defaultCycleColors {
```

Comment:
We also do this stuff with static functions. Would be good to have a best practice defined.